noun/manage.c

```diff
@@ -1708,13 +1717,39 @@ u3m_boot(c3_o nuu_o, c3_o bug_o, c3_c* dir_c,
     printf("boot: loading %s\r\n", ful_c);
 
     {
-      u3_noun sys = u3ke_cue(u3m_file(ful_c));
-      u3_noun bot;
+      u3_noun pil = u3m_file(ful_c);
+      u3_noun sys, bot;
+
+      {
+        u3_noun pro = u3m_soft(0, u3ke_cue, u3k(pil));
+
+        if ( 0 != u3h(pro) ) {
```

Comment:
Is it ok to use a raw zero here, instead of `u3_blit` or whatever?

Reply:
I don't think there's a more specific convention for checking the tag of a bail noun (a `+tone` or  `+toon`). The success case is `[%0 p=*]`, so there won't be any practical issue.

Reply:
Ah, it's a noun zero. Ok, that's fine.